pk/ecc/ecc_sign_deterministic.c

```diff
@@ -0,0 +1,530 @@
+/* ecc_sign_deterministic.c
```

Comment:
Please exclude the compiled binary from the comment `pk/ecc/ecc_sign_deterministic` and also add to the `wolfssl-examples/.gitignore`